modules/standard/BigInteger.chpl

```diff
@@ -3945,12 +3841,41 @@ module BigInteger {
     }
   }
 
-  @deprecated(notes="bigint.divR method is deprecated - please use the standalone function :proc:`~BigInteger.divR`")
+  /* See :proc:`~BigInteger.div` */
+  proc div(ref       result: bigint,
+           const ref numer: bigint,
+                     denom: integral,
+           param     rounding = roundingMode.zero)
+    do BigInteger.div(result, numer, new bigint(denom), rounding);
+
+  /* Divide ``numer`` by ``denom``, forming a remainder and storing it in
+     ``this``.  The absolute value of the remainder will always be less than the
+     absolute value of the denominator (i.e. ``abs(this) < abs(denom)``).
+
+     :arg numer: The numerator of the division operation to be performed
+     :type numer: :record:`bigint`
+
+     :arg denom: The denominator of the division operation to be performed
+     :type denom: :record:`bigint`, ``integral``
+
+     :arg rounding: The rounding style to use, see :enum:`round` for a
+                    description of what the rounding styles entail.  Defaults to
+                    ``zero`` if unspecified
+     :type rounding: ``round``
+
+     .. warning::
+        If the denominator is zero, the program behavior is undefined.
+  */
+  @deprecated(notes="bigint.divR method is deprecated - please use the standalone function :proc:`~BigInteger.rem` with :enum:`roundingMode` instead")
+  proc bigint.divR(const ref numer: bigint,
+                             denom: integral,
+                   param     rounding = round.zero)
+    do BigInteger.rem(this, numer, denom, chpl_roundToRoundingMode(rounding));
+  @deprecated(notes="bigint.divR method is deprecated - please use the standalone function :proc:`~BigInteger.rem` with :enum:`roundingMode` instead")
```

Comment:
Very minor, but maybe add a space between this function and the next. 

I think this gets more confusing with the `do` syntax for one line functions, which is one of the reasons I'm not the biggest fan of that syntax, but I won't object to that being used.

Reply:
I definitely agree on the spacing, I will add new lines.

For the `do` syntax, I prefer it in these cases where its just forwarding the different function (like a renamed function for deprecation). But I wouldn't be opposed to just reverting to curly braces

Reply:
I don't think you should remove the `do` syntax. It isn't my favorite, but I don't have a problem with others using it.